Build/libHttpClient.Apple.C/libHttpClient.xcodeproj/project.pbxproj

```diff
@@ -216,8 +216,8 @@
 			isa = PBXGroup;
 			children = (
 				58A7E982209ADEB100CC6774 /* mock_publics.cpp */,
```

Comment:
Nit: The order of these dictates the order that Xcode shows them in the editor. alphabetize these please.